frontend/packages/kubevirt-plugin/integration-tests-cypress/tests/disks/hotplug.spec.ts

```diff
@@ -0,0 +1,271 @@
+import { DISK_SOURCE, TEMPLATE_NAME, VM_ACTION_TIMEOUT, VM_STATUS, VM_ACTION } from '../../const';
+import { ProvisionSource } from '../../enums/provisionSource';
+import { testName } from '../../support';
+import { Disk, VirtualMachineData } from '../../types/vm';
+import { confirmButton, detailViewAction } from '../../view/actions';
+import { addDisk, delDisk, waitForCurrentVMStatus } from '../../view/dialog';
+import * as tags from '../../view/selector';
+import { tab } from '../../view/tab';
+import { virtualization } from '../../view/virtualization';
+import { vm, waitForStatus } from '../../view/vm';
+
+const vmData: VirtualMachineData = {
+  name: `test-vm-hotplug-${testName.split('-')[1]}`,
+  namespace: testName,
+  template: TEMPLATE_NAME,
+  provisionSource: ProvisionSource.URL,
+  pvcSize: '1',
+  sshEnable: false,
+  startOnCreation: true,
+};
+
+const hotPlugDisk: Disk = {
+  size: '2',
+};
+
+const FCB32_URL_IMG =
+  'http://cnv-qe-server.rhevdev.lab.eng.rdu2.redhat.com/files/cnv-tests/fedora-images/Fedora-Cloud-Base-32-1.6.x86_64.qcow2';
+const FCB32_URL_REG = 'quay.io/kubevirt/fedora-cloud-container-disk-demo:latest';
+const PVC_NAME = 'hotplug-test-pvc';
+
+describe('Test UI for VM disk hot-plug', () => {
+  before(() => {
+    cy.Login();
+    cy.visit('/');
+    cy.createProject(testName);
+    vm.create(vmData);
+    waitForStatus(VM_STATUS.Running, vmData, VM_ACTION_TIMEOUT.VM_IMPORT);
+    virtualization.vms.visit();
+    cy.byLegacyTestID(vmData.name)
+      .should('exist')
+      .click();
+    tab.navigateToDisk();
+  });
+
+  after(() => {
+    cy.deleteResource({
+      kind: 'VirtualMachine',
+      metadata: {
+        name: vmData.name,
+        namespace: vmData.namespace,
+      },
+    });
+    cy.deleteResource({
+      kind: 'Namespace',
+      metadata: {
+        name: testName,
+      },
+    });
+  });
+
+  it('ID(CNV-6828) Attach Persistent hotplug disk with [Blank] as source to running VM', () => {
```

Comment:
There are a few tests like this with different disk settings, I think use a loop here could make it more simple and maintainable. Example: https://github.com/openshift/console/blob/2f9a41c05206312e0d566cb63c079f0a8a50a921/frontend/packages/kubevirt-plugin/integration-tests-cypress/tests/vm/advanced-wizard.ts#L90

Reply:
I thought of it, but then all the removal tests won't benefit from already added disks

Reply:
Ideally, the tests should depend on previous tests. 
https://docs.cypress.io/guides/references/best-practices#Having-tests-rely-on-the-state-of-previous-tests

IMHO, Use a loop for the similar tests make it easier to read/maintain.